### DISCUSSION TITLE
Modal constrains size of searchable dropdown select element

### DISCUSSION THREAD

**Discussion by Relequestual** [community/community#39435] [Accessibility]
### Select Topic Area

Bug

### Body

Modal constrains size of searchable dropdown select element.
(I contacted support and was redirected here)

Reproduction Steps:
Try to add a repo to a team of an org.

Fault Type:
Permanent - Occurring repeatably

Blocking?:
No

![image](https://user-images.githubusercontent.com/731158/202741863-42e20d27-2c64-4a9c-93ae-48c7d2b334b0.png)

**Comment by cehfisher** [comment]
Hi @Relequestual - I wanted to follow up on this stale issue. When trying to replicate this issue, it looks like the modal from this workflow was removed. Could you see if these workflow changes resolved the original issue for you? If yes, could you please close this issue? If no, what additional changes are needed? Please supply new screenshots and URLs, if possible. Thank you! (cc @queenofcorgis)
